models/FeaturedItemsSection.yaml

```diff
@@ -0,0 +1,121 @@
+type: object
+name: FeaturedItemsSection
+label: Featured Items section
+labelField: title
+extends:
+  - Section
+groups:
+  - sectionComponent
+fieldGroups:
+  - name: styles
+    label: Styles
+  - name: settings
+    label: Settings
+fields:
+  - type: string
```

Comment:
Missing `elementId` field

Reply:
I think it's inherited from the section yaml.

Reply:
Yes I think you are right. No need to add it then.